### DISCUSSION TITLE
Lesson 6 Foundry installation error

### DISCUSSION THREAD

**Discussion by NeoRusi** [Cyfrin/foundry-full-course-cu#729] [Q&A] [answered]
I am on lesson 6( I am using MACOS) and I get up this output after running
```
curl -L https://foundry.paradigm.xyz | bash
```
```
warning: libusb not found. You may need to install it manually on MacOS via Homebrew (brew install libusb).
```
Maybe someone nows what is libusb ?

**Comment by theirrationalone** [comment]
Please install `libusb` dev dependency... 

```terminal
sudo apt-get install libusb-1.0-0-dev
```

And then try again....

**Reply by NeoRusi** [reply]
Thanks but for some reason I can't sign the password( when I am typing it doesn't responds) needed to install this dependency

**Reply by theirrationalone** [reply]
Please type your password and don't care, Password will remain invisible during entering..., After entering right password hit `Enter Key`

**Reply by NeoRusi** [reply]
now it tells this 
```
sudo: apt-get: command not found
```

**Reply by theirrationalone** [reply]
You need to install homebrew:

Try this [documentation](https://docs.brew.sh/Installation)

**Reply by theirrationalone** [reply]
Or if you're facing problems on above link... try this [homebrew macos](https://brew.sh/)

**Reply by theirrationalone** [reply, marked as answer]
After installing homebrew, Try these commands:

install:
```terminal
brew install libusb
```

info:
```terminal
brew info libusb
```

after this mitigation you can use that curl command to install foundry!

**Reply by NeoRusi** [reply]
Hey bro how didi you now what should I do ?

**Reply by theirrationalone** [reply]
Actually your Terminal's error was indicating me from the beginning😉
